doc/installation.rst

```diff
@@ -0,0 +1,27 @@
+Installation
+============
+1. Install this package:
+
+   .. code-block:: console
+
+       python setup.py install
+
+2. Modify your Swift :file:`object-server.conf` to use the new object server:
+
+   .. code-block:: ini
+
+       [app:object-server]
+       use = egg:swift_scality_backend#sproxyd_object
+
+3. Set the Sproxy host(s) and Sproxy path to connect to in the
+   ``[app:object-server]`` section in the same file. If your system supports it
```

Comment:
We just repeat the file name, again. I'd replace `in the same file` with :file:`object-server.conf`